src/pages/atomicdex/api/v20-dev/lightning/channels/index.mdx

```diff
@@ -0,0 +1,924 @@
+        "coin": "tBTC-lightning",
+        "node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735",
+        "amount": {
+            "type": "Exact",
+            "value": 0.004
+        }
+    },
+    "id": 22
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"uuid": "434681f8-95e5-484f-af49-fa80d8ae857b",
+		"node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@233.112.95.142:9735"
+	},
+	"id": 22
+}
+```
+</CollapsibleSection>
+
+
+#### 📌 Example with all available configuration options
+
+<CodeGroup title="" tag="POST" label="METHOD_NAME">
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::open_channel",
+    "params": {
+        "coin": "tBTC-lightning",
+        "node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735",
+        "amount": {
+            "type": "Max"
+        },
+        "push_msat": 0,
+        "channel_options": {
+            "proportional_fee_in_millionths_sats": 0,
+            "base_fee_msat": 1000,
+            "cltv_expiry_delta": 72,
+            "max_dust_htlc_exposure_msat": 5000000,
+            "force_close_avoidance_max_fee_satoshis": 1000
+        },
+        "channel_configs" : {
+            "counterparty_locktime": 144,
+            "our_htlc_minimum_msat": 1,
+            "negotiate_scid_privacy": false,
+            "max_inbound_in_flight_htlc_percent": 10,
+            "announced_channel": false,
+            "commit_upfront_shutdown_pubkey": true
+        }
+    },
+    "id": null
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"uuid": "2b50e274-c173-4fa1-95f3-97f9f82ace92",
+		"node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735"
+	},
+	"id": null
+}
+```
+</CollapsibleSection>
+
+
+## Close a Channel {{label : 'lightning::channels::close_channel', tag : 'API-v2'}}
+
+The `lightning::channels::close_channel` method closes a channel with a remote node.
+
+### Request Parameters
+
+| Parameter         | Type    | Description                                                                              |
+|-------------------|---------|------------------------------------------------------------------------------------------|
+| coin              | string  | The ticker of the coin you would like to view closed channels for.                       |
+| uuid              | string  | Then channel UUID.                                                                       |
+| force_close       | boolean | Optional, defaults to `false`. If `true`, will force closure of the channel.             |
```

Comment:
Please note that it's not advised to force close the channel except if the counterparty is offline for sometime and is not reachable. Force closure of a channel will make the party who does the force closure to wait for a number of blocks equal to `force_close_spend_delay` before they can get their funds on chain.

@smk762 `force_close_spend_delay` is not available in `get_channel_details` response, I should add it there and also add it to the response here.

Reply:
Is `force_close_spend_delay` configurable by the user? I have no other references to it in the docs yet.

Reply:
It's configured by the other side using `counterparty_locktime` where we are the counterparty to the other side, the other side gets to set it since it's part of their security considerations since they have to be online at least once during this period if we tried to steal funds by broadcasting an old commitment transaction. We set our own too in our configs where it shows in the channel details in the other side as `force_close_spend_delay`. The other side can't set `counterparty_locktime` to any value when opening the channel though, since they can set it to a very long time and our funds will be locked for a very long time if we force closed the channel, this is where `counterparty_channel_config_limits::our_locktime_limit` come in as the other side can't make `counterparty_locktime` larger than this limit otherwise the channel gets rejected and is never opened in the first place.